actionview/lib/action_view/helpers/csp_helper.rb

```diff
@@ -4,6 +4,9 @@ module ActionView
   module Helpers # :nodoc:
     # = Action View CSP \Helpers
     module CspHelper
+      mattr_accessor :rename_csp_helper_nonce_attribute
+      self.rename_csp_helper_nonce_attribute = nil
```

Comment:
Maybe things could be a bit simpler if we make the attribute name the setting:
```suggestion
      mattr_accessor :csp_meta_tag_nonce_attribute, default: :content
```

Reply:
If there's still interest in merging this in all related projects I'll take a look and change it 👍

Reply:
I think we will see activity on those related project PRs once this change lands.